gpkit/nomials/nomial_math.py

```diff
@@ -1,20 +1,19 @@
 """Signomial, Posynomial, Monomial, Constraint, & MonoEQCOnstraint classes"""
```

Comment:
potentially? core and math should definitely be renamed, but I don't think their current names are very accurate or useful.